nxt_editor/dockwidgets/output_log.py

```diff
@@ -334,7 +334,7 @@ def write_raw_output(self, val, msg_time=0.):
                 curr_rt_layer.cache_layer.was_during_node_exec(msg_time)):
             return
         if self.log_filter_button.is_level_enabled(nxt_log.NODEOUT):
-            self.write_rich_output(val)
+            self.write_rich_output(val.replace('<', '&lt;'), raw=True)
```

Comment:
was `raw=True` a leftover from an old approach?

Also would be nice to write a lil docstring about why we're doing this.

Reply:
whoops, yeah. I'll fix that and add a note.